### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by myfriendgu** [CivicActions/fema-grants#71]

**Comment by myfriendgu** [comment]
Testing artifacts here: https://drive.google.com/open?id=1P0qmCEGDXSnSW0YvF7-gPoFTaOAkCeAE

A couple notes:

- I tested the dashboard, create report, and view individual report page. 

- You can view the Google Lighthouse report files from Google here: https://googlechrome.github.io/lighthouse/viewer/ There's some good information on accessibility and where things failed 

- Included screen captures of me using a screenreader. By far the biggest thing is not having everything readable in the table elements that we have, especially in the individual report page. This and labeling each element correctly seem to be the biggest blockers to accessibility.
